docs/windows.md

```diff
@@ -0,0 +1,195 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports to running on Windows worker Node, and setups an overlay network
+which uses VXLAN tunneling to route packets between nodes.
+
+One HNS Network with [Transparent mode](https://docs.microsoft.com/en-us/virtualization/windowscontainers/container-networking/network-drivers-topologies)
+is created on a Windows worker node, on which the [Open vSwitch Extension](http://docs.openvswitch.org/en/latest/intro/install/windows)
+is enabled. The HNS Network connects directly to the physical network through the network adapter whose IP address was used as the
+Node IP when the Windows Node joined the cluster. An OVS bridge is created, and leverages the network adapter as
+the uplink interface. The original IP and MAC address of the network adapter are moved to the OVS bridge.
+
+One HNS Endpoint is created on this HNS Network and attached on all the containers
+in same Pod when the CNI ADD command is received. An internal OVS port is created
+at the same time, which has the same name as the HNS Endpoint. OpenFlow entries
+are installed on the OVS bridge to provide container networking.
+
+Each Windows Node is assigned a single subnet, and 
+[host-local IPAM plugin](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/host-local)
+is invoked to allocate IPs from the subnet to all local Pods.
+
+### Components that run on Windows
+
+The following components should be configured and run on the Windows Node.
+* [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
+* OVS daemons
+* Antrea Agent
+* Kube-proxy
+
+Antrea-Agent and Kube-proxy run as processes on host and are managed by management Pods. It is recommended to run OVS
+daemons as Windows services.
+
+### Traffic walk
+* ***Intra-Node traffic*** and ***Inter-Node traffic*** On Windows Node, the
+forwarding path for packets that are between two local Pods or to a Pod on
+another Node is the same as Linux Node. Please refer to the [architecture](/docs/architecture.md).
+
+* ***Pod to external traffic*** Antrea Agent creates OpenFlow rules to perform
+SNAT on the packets sent from local Pod to an external IP or the Nodes' network,
+and the source IP will be rewritten to the Node's IP. IP-Forwarding is enabled on the OVS bridge local adapter to
+forwarding the traffic.
+
+## Deploying Antrea on Windows Worker Node
```

Comment:
I would put this part before implementation/architecture (if you do not want to create a separate user doc for installation and usage).

Reply:
Wenying have a detiled design doc, I think I can migrate the doc to a separate doc in this repo.

Reply:
That sounds good to me. If we are going to have a separate design/architecture doc, I suggest to focus on installation&usage for this PR, and merge it first.